aip/general/0193.md

```diff
@@ -69,13 +69,27 @@ and the method **should** put partial failure information in the metadata
 message. The errors themselves **must** still be represented with a
 [`google.rpc.Status`][Status] object.
 
+### Permission Denied
```

Comment:
quick note: I think this guidance is actually stronger than the guidance that existed in the original AIPs (since this is more general guidance on all operations, not just resource-oriented ones).

But I think the spirit is the same. Happy to reduce this in scope if we want to discuss the delta in more granular detail.